base/applications/winver/winver.c

```diff
@@ -4,17 +4,25 @@
  * FILE:            base/applications/winver/winver.c
  */
 
+#define WIN32_LEAN_AND_MEAN 1
 #include <stdarg.h>
-#include <windef.h>
-#include <winbase.h>
+#include <windows.h>
```

Comment:
Why including the whole header?

Reply:
because making the same thing with WIN32_LEAN_AND_MEAN was easier.
If it's a problem I will replace that.

Reply:
We try to include explicitly the very bare minimum number of headers so as to not trigger unnecessary recompiles in case a possibly unrelated header (that here would be included by windows.h but completely unnecessary for winver) gets changed.
So, if windef+winbase inclusion still works, then please keep them instead.